src/hooks/use-render.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Do you know why these could now be removed?

Reply:
That disable isn't needed on main either. I think that is a leftover from some other refactoring